src/journal/lookup3.h

```diff
@@ -1,3 +1,5 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
```

Comment:
similar, "Public Domain"

Reply:
There is no "Public domain" license, see the commit message.

Reply:
hmm? there is no commit msg on this commit?

Reply:
maybe drop the spdx tg for this file then?

I found some other projects importing the jenkins hash with spdx header:

https://github.com/DPDK/dpdk/blob/main/lib/librte_hash/rte_jhash.h ← claims it was BSD licensed
https://github.com/OpenImageIO/oiio/blob/master/src/include/OpenImageIO/hash.h ← ditto

Qt just glues its own license on top:

https://sources.debian.org/src/qtbase-opensource-src/5.14.2+dfsg-6/tests/baselineserver/shared/lookup3.cpp/?hl=34#L34

Maybe we should use the SPDX LicenseRef stuff for this, i.e.

```
/* SPX-License-Identifier: LicenseRef=lookup3-public-domain */
```

if i grok the spdx spec right this is how we can denote "made up" licenses